<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Ã©tudiants</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 62, 89);
border-right-color: rgb(44, 62, 89);
border-color: rgb(44, 62, 89);
selection-background-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-bottom-color: rgb(44, 62, 89);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des Achats</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un achat</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num_Achat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_Achat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer un achat</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num_Achat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les achats</string>
       </attribute>
       <widget class="QTableView" name="tabachat">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_achat">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Rechercher achat</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>num_achat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rechac">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabrechac">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>221</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Modifier un achat</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> Date_Achat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_mod">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>num_achat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Gestion des vents</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Ajouter une vent</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num_vente</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_vente</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Supprimer une vente</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num_vente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Afficher les ventes</string>
       </attribute>
       <widget class="QTableView" name="tabvente">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Rechercher vents</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>NUM_VENTE</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rechv">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> 
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>RECHERCHE</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabrechv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>181</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Modifier une vente</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>  Date_Vente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modve">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>num_vente</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
